packages/release-please/test/release-please.ts

```diff
@@ -757,40 +834,4 @@ describe('ReleasePleaseBot', () => {
       });
     });
   });
-
-  it('should fetch default branch from GitHub if not provided', async () => {
-    const payload = require(resolve(fixturesPath, './push_to_main'));
```

Comment:
Do we have anythiung that confirms we default to the appropriate branch if not config provided? Is there a test for this in release-please itself?